packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -195,7 +225,8 @@ function Picker<T extends object>(props: PickerProps<T>, ref: FocusableRef<HTMLB
     necessityIndicator,
     UNSAFE_className = '',
     UNSAFE_style,
-    placeholder = 'Select an option...',
+    placeholder = 'Select...',
```

Comment:
based on https://github.com/adobe/react-spectrum/pull/6796

can change it back if we want "Select an option" for S2

Reply:
Should be the short version, thanks